CONTRIBUTING.md

```diff
@@ -1,26 +1,16 @@
-# Web Media Text Tracks Community Group
+# Contributor License Agreement
 
-This repository is being used for work in the Web Media Text Tracks Community Group, governed by the [W3C Community License
-Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/). To contribute, you must join
```

Comment:
I disagree with removing the CLA and CG reference. It encourages people that cannot join the W3C to participate.

Reply:
As a W3C specification on the REC track, I do not see why WebVTT should be subject to a CLA different than other W3C specifications on the REC track. The CG can create a separate repo if it wants to work on something other then the specification on the REC track.